src/mumble/ManualPlugin.h

```diff
@@ -20,7 +20,11 @@
 #include "ui_ManualPlugin.h"
 
 #ifdef Q_OS_UNIX
+#define __cdecl
 typedef WId HWND;
+#define DLL_PUBLIC __attribute__((visibility("default")))
+#else
+#define DLL_PUBLIC __declspec(dllexport)
```

Comment:
[mumble_plugin.h](https://github.com/mumble-voip/mumble/blob/master/plugins/mumble_plugin.h) I don’t see it. No export at all.

(I do see a lot of `__cdecl`s though.)